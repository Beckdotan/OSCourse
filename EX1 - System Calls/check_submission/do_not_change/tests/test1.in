V
4096 aux_files/from1.txt aux_files/to1.txt
