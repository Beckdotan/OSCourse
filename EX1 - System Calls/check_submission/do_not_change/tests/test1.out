Content from file aux_files/from1.txt was successfully appended to aux_files/to1.txt
